claude_mcp.sh

```
@@ -554,10 +560,10 @@ if EXISTING_SERVERS=$(claude mcp list 2>&1); then
 
     if [ "$EXISTING_COUNT" -gt 0 ]; then
         echo -e "${BLUE}📋 Running parallel health checks on existing servers...${NC}"
-        
+
         # Extract server names and start parallel health checks
         SERVER_NAMES=($(echo "$EXISTING_SERVERS" | grep -E "^[a-zA-Z].*:" | cut -d':' -f1))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)